pkgs/desktops/gnome-3/3.22/misc/gnome-tweak-tool/default.nix

```diff
@@ -14,6 +14,10 @@ in stdenv.mkDerivation rec {
 
   makeFlags = [ "DESTDIR=/" ];
 
+  preConfigure = ''
```

Comment:
`PYTHON = python.interpreter;` instead of this preConfigure step is a bit nicer if that works.
And a comment explaining why this part is here would also be nice for people that look at this in the future.